### PR TITLE
Enable running tests in parallel.

### PR DESCRIPTION
**Pull request by Tapchicoma**
Should speedup plugin test suite execution time.

### CODE REVIEW

**Review by JLLeitschuh — COMMENTED** [review]
LGTM!

**Review by JLLeitschuh — APPROVED** [review]

**Comment by Tapchicoma** [comment]
Wanted to add TestKit caching on Github actions before merging it

**Comment by Tapchicoma** [comment]
Doesn't seem to help a lot with Github actions (seems it uses low CPU runners for opensource), but, at least, tests are running faster locally :man_shrugging:

File: plugin/src/test/kotlin/org/jlleitschuh/gradle/ktlint/AbstractPluginTest.kt [hunk 1]
```diff
@@ -55,11 +55,20 @@ abstract class AbstractPluginTest {
...
         projectRoot: File = this.projectRoot
-    ): GradleRunner =
-        GradleRunner.create()
+    ): GradleRunner {
+        val sharedTestKitDir = File(".")
+            .resolve(".gradle-test-kit")
+            .absoluteFile
+            .also {
+                if (!it.exists()) it.mkdir()
+            }
```
Comment (JLLeitschuh) [inline plugin/src/test/kotlin/org/jlleitschuh/gradle/ktlint/AbstractPluginTest.kt:64]: Theoretically, this won't need in Gradle 7.0

**Pull request merged**